terminus-terminal/package.json

```diff
@@ -34,12 +36,12 @@
     "rxjs": "5.3.0"
   },
   "dependencies": {
-    "font-manager": "0.2.2",
```

Comment:
Please revert changes that affect dependency versions. Changes to formatting are fine though!